<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1578</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Strategy calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QGroupBox" name="testDates">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>801</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Dates to test on</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_run">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>240</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>421</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="results_2">
       <item>
        <widget class="QLabel" name="residual_risk">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="result_residual_risk">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="risk_reduction">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="result_risk_reduction">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>335</width>
        <height>83</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="input_grid">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The number of days ago that the patient has been infected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Time passed since infection</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox_known">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>known</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="input_timepassed">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>14.000000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_15">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>days</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox_unknown">
         <property name="text">
          <string>unkown</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Quarantaine duration</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="input_quarantaine">
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>21.000000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>days</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Parameters</string>
     </attribute>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The sensitivity of the PCR test.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>PCR sensitivity</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="param_pcr_sens">
      <property name="geometry">
       <rect>
        <x>313</x>
        <y>210</y>
        <width>74</width>
        <height>24</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;P(PCR+|inf)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>70.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>393</x>
        <y>210</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>241</y>
        <width>79</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The specificity of the PCR test.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>PCR specifity</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="param_pcr_spec">
      <property name="geometry">
       <rect>
        <x>313</x>
        <y>241</y>
        <width>74</width>
        <height>24</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;P(PCR-|not inf)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>99.500000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>393</x>
        <y>241</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>22</y>
        <width>495</width>
        <height>116</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_26">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Mean duration of pre-detectability phase; PCR-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="param_time_predetect">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>99.989999999999995</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>days</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_28">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Mean duration presymptomatic; PCR+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="param_time_presymp">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>days</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_30">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Mean duration infectious, (symptomatic); PCR+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="param_time_infect">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>4.810000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>days</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Mean duration post-infectious; PCR+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="param_time_post">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.900000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>days</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>393</x>
        <y>161</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>161</y>
        <width>228</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Percentage of asymtomatic infections</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="param_perc_asym">
      <property name="geometry">
       <rect>
        <x>313</x>
        <y>161</y>
        <width>74</width>
        <height>24</height>
       </rect>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="results" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>1561</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="result_log" native="true">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>731</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="actionEdit_and_View">
   <property name="text">
    <string>Edit and View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
